futures-util/Cargo.toml

```diff
@@ -34,6 +34,7 @@ unstable = ["futures-core-preview/unstable"]
 cfg-target-has-atomic = ["futures-core-preview/cfg-target-has-atomic"]
 bilock = []
 read_initializer = ["io", "futures-io-preview/read_initializer", "futures-io-preview/unstable"]
+write_all_vectored = ["io"]
```

Comment:
Can you add a corresponding feature to the top-level `futures` crate which turns this on, as well as a conditional re-export of the public types introduced?